src/elm/TreeUtils.elm

```diff
@@ -127,8 +135,7 @@ getNext id tree =
 getPrevNextInColumn : Int -> String -> Tree -> Maybe Tree
 getPrevNextInColumn shift id tree =
```

Comment:
This is a good reference for a new `gotoTopOfColumn` function.
Here one approach to get the first/last `Tree` (aka card) in the column:
- [ ] use `getColumnById` to get the current column (type: `List ( List Tree )`)
- [ ] use `List.concat` to flatten that into `List Tree`
- [ ] use `List.map .id` to apply `.id` to each `Tree`, extracting the `id` field (type is now: `List String`)

Then `List.head` (with or without `List.reverse`) will give you the id of the first/last card in the column.